sherpa/conftest.py

```diff
@@ -82,6 +82,7 @@ def pytest_configure(config):
     """
     try:
         path = config.getoption(TEST_DATA_OPTION)
-        SherpaTestCase.datadir = path
+        if (path):
```

Comment:
A quick style question: are the brackets around `path` meaningful here? I've generally been removing them when I see them in the code base as I find they just add visual clutter, but perhaps there's a reason for them I'm missing!